src/main/resources/db/migration/V5.13.0.3__8889-change-filepids-default.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO setting(content, lang, name) VALUES
+              ('true', '', ':FilePIDsEnabled')
```

Comment:
```suggestion
              ('true', NULL, ':FilePIDsEnabled')
```

Is the above what we want instead? NULL instead of empty string? With empty string I'm getting the error below.